plugins/chat/assets/javascripts/discourse/services/chat-api.js

```diff
@@ -29,6 +30,21 @@ export default class ChatApi extends Service {
     );
   }
 
+  /**
+   * Get a thread by its ID.
+   * @param {number} threadId - The ID of the thread.
+   * @returns {Promise}
+   *
+   * @example
+   *
+   *    this.chatApi.thread(1).then(thread => { ... })
+   */
+  thread(threadId) {
+    return this.#getRequest(`/threads/${threadId}`).then((result) =>
+      this.chatThreadsManager.store(result.thread)
+    );
+  }
```

Comment:
Maybe you have a good reason for this, but my initial reaction would be that thread should always have to be called under `/channels/:channel_id/threads/:thread_id` We will have to check thread is part of a channel anyways for security, so better enforce it at the route level IMO.

Reply:
Yeah I did toss up whether to do this or not, my only thought was that it may be nice to be able to call this route with only a thread ID if you don't have the channel ID? But if you think that's not useful, will change it to be this way.

Reply:
Im not totally against it, but if you don't have strong feelings against it, I would prefer we have it under channel for now.

Reply:
Sure, will change!